docs/userGuide/deployingTheSite.md

```diff
@@ -54,16 +54,17 @@ You can override the default deployment settings %%(e.g., repo/branch to deploy)
 
 Here are the steps to set up Travis CI:
 
-1. [Sign up with GitHub](https://travis-ci.com/signin) at [Travis CI](https://travis-ci.com).
+1. [Sign in to Travis](https://travis-ci.com/signin) using your GitHub account.
 1. Accept the authorisation for Travis CI when you are redirected to GitHub.
-1. Click the green _Activate_ button, and select the repository with the MarkBind site.
+1. Go to the [Repositories page](https://travis-ci.org/account/repositories).
+1. Find the repository with the MarkBind site.<br>
+   %%{{ icon_info }} If the organization/repository is not shown in the list, click on `Review and add` link at the bottom of the list of organization and follow the steps to give Travis access to the organization containing your repo. After that, come back to the [Repositories page](https://travis-ci.org/account/repositories) and use the `Sync Account` button to sync your Travis account with GitHub.%%
+1. Activate the repo using the slider switch in front of it.
```

Comment:
This set of instructions is for Legacy Services Integration on `travis-ci.org`, which is currently [undergoing migration to `travis-ci.com`](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/). To summarise the pertinent changes:
- [new open source repositories should be added on `travis-ci.com` instead of `travis-ci.org`.](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/#new-repositories)
- new repositories on `travis-ci.com` will use GitHub Apps integration instead of the legacy OAuth integration (which is what this set of instructions are using)
- [existing open source repositories on `travis-ci.org` will remain, although there is a closed beta to migrate these repositories to `travis-ci.com` as well](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/#existing-open-source-repositories-on-travis-ciorg). We should assume that there will only be `travis-ci.com` in the future.

The flow given in the previous version was assuming users were adding the Travis CI GitHub app to their account for the first time. Here's what that flow should look like:
1. [Sign in to Travis](https://travis-ci.com/signin) using your GitHub account.
![](https://user-images.githubusercontent.com/7784660/42061902-fed5be72-7b2b-11e8-9f70-7c4eec7e1807.jpg)
1. Accept the authorisation for Travis CI when you are redirected to GitHub.
![](https://user-images.githubusercontent.com/7784660/42060974-e6384036-7b28-11e8-9aa1-1535dabe0dee.jpg)
1. Click the green _Activate_ button, and select the repository with the MarkBind site.
![](https://user-images.githubusercontent.com/7784660/42061901-feb97028-7b2b-11e8-9ac5-75e32a181087.jpg)
![](https://user-images.githubusercontent.com/7784660/42060973-e61fe702-7b28-11e8-8e40-f99e26281750.jpg)

Since the documentation was written from the perspective of a first time Travis CI user, I thought it would be better if we captured the new Travis CI GitHub App flow. However, I do understand that there will be many open source projects that are currently using `travis-ci.org` and the legacy service integration. 

Perhaps we should include both sets of instructions, with the appropriate caveats, until Travis CI sorts out the migration?

Reply:
> Perhaps we should include both sets of instructions, with the appropriate caveats, until Travis CI sorts out the migration?

I wasn't aware of the Travis migration to a new system. Yes I guess we should include both sets of instructions for the time being. Can you take over the PR and add it in?

Reply:
@damithc added!